simple/tests/stats/test_data/nodes/expected/triples.csv

```diff
@@ -0,0 +1,53 @@
+subject_id,predicate,object_id,object_value
+custom/s/Source_default,typeOf,Source,
+custom/s/Source_default,name,,Custom Data Commons
+custom/s/Source_1,typeOf,Source,
+custom/s/Source_1,name,,Source1
+custom/s/Source_1,url,,http://source1.com
+custom/s/Source_1,domain,,source1.com
+custom/base/Prov_1,typeOf,Provenance,
+custom/base/Prov_1,name,,Provenance1
+custom/base/Prov_1,source,custom/s/Source_1,
+custom/base/Prov_1,url,,http://source1.com/provenance1
+custom/base/Prov_2,typeOf,Provenance,
+custom/base/Prov_2,name,,b.csv
+custom/base/Prov_2,source,custom/s/Source_default,
+custom/base/Prov_3,typeOf,Provenance,
+custom/base/Prov_3,name,,x.csv
+custom/base/Prov_3,source,custom/s/Source_default,
+custom/g/group_1,typeOf,StatVarGroup,
+custom/g/group_1,name,,Parent Group
+custom/g/group_1,specializationOf,dc/g/Root,
+custom/g/group_2,typeOf,StatVarGroup,
+custom/g/group_2,name,,Child Group 1
+custom/g/group_2,specializationOf,custom/g/group_1,
+custom/g/group_3,typeOf,StatVarGroup,
+custom/g/group_3,name,,Child Group 2
+custom/g/group_3,specializationOf,custom/g/group_1,
+custom/g/Root,typeOf,StatVarGroup,
+custom/g/Root,name,,Custom Variables
+custom/g/Root,specializationOf,dc/g/Root,
+custom/statvar_1,typeOf,StatisticalVariable,
+custom/statvar_1,name,,Variable 1
+custom/statvar_1,memberOf,custom/g/group_2,
+custom/statvar_1,includedIn,custom/base/Prov_1,
+custom/statvar_1,includedIn,custom/s/Source_1,
+custom/statvar_2,typeOf,StatisticalVariable,
+custom/statvar_2,name,,Variable 2
+custom/statvar_2,memberOf,custom/g/group_2,
+custom/statvar_2,includedIn,custom/base/Prov_1,
+custom/statvar_2,includedIn,custom/s/Source_1,
```

Comment:
Do we need to have SV --> source too?  Given you can get to it from provenance?

Reply:
@shifucun?

Reply:
Yes, we have <sv, source> existence cache generated and it is used in the stat var explorer. For example:

`./check_bt d/2/dc/s/UsCensusBureau^Count_Person infrequent | ./cache_parse`

Reply:
sg